packages/core/src/config.ts

```diff
@@ -73,7 +73,7 @@ export function resolveConfig(
     mergeSelectors: true,
     warn: true,
     blocklist: [],
-    safelist: [],
+    safelist: mergePresets('safelist'),
```

Comment:
You might need to move this under `...config`

Reply:
Ah right, I missed that part thanks.